x-pack/plugins/security_solution/server/lib/detection_engine/prebuilt_rules/api/perform_rule_upgrade/perform_rule_upgrade_route.ts

```diff
@@ -143,10 +144,14 @@ export const performRuleUpgradeRoute = (router: SecuritySolutionPluginRouter) =>
                   });
                 }
                 break;
-              case PickVersionValues.CURRENT:
+              case PickVersionValuesEnum.CURRENT:
                 targetRules.push({ ...current, version: target.version });
                 break;
-              case PickVersionValues.TARGET:
+              case PickVersionValuesEnum.TARGET:
+                targetRules.push(target);
+                break;
+              case PickVersionValuesEnum.MERGED:
+                // TODO: Implement functionality to handle MERGED
```

Comment:
Since this is an internal endpoint, and we don't use `MERGED` yet, this should be safe. Defaulting to updating to `TARGET` anyways, until actual endpoint implementation is finished.